core/protocol/all_services.go

```diff
@@ -198,6 +201,15 @@ func newServices(
 	svcs.protocolUpgradeEngine = protocolupgrade.New(svcs.log, svcs.conf.ProtocolUpgrade, svcs.broker, svcs.topology, version.Get())
 	svcs.witness = validators.NewWitness(svcs.log, svcs.conf.Validators, svcs.topology, svcs.commander, svcs.timeService)
 
+	specEngine, err := ethcall.NewEngine(svcs.log, ethcall.Config{}, nil, svcs.eventForwarder)
+	if err != nil {
+		svcs.log.Error("unable to initialise eth call engine", logging.Error(err))
+		return nil, err
+	}
+
+	svcs.ethereumOraclesVerifier = oracles.NewEthereumOracleVerifier(svcs.log, svcs.witness, svcs.timeService, svcs.oracle,
```

Comment:
I think you need to add this engine to `registerTimeServiceCallbacks()` for the `OnTick()` callback to happen.